src/classes/RD2_DataMigration_TEST.cls

```diff
@@ -2370,4 +2446,25 @@ private class RD2_DataMigration_TEST {
         return Database.insert(new Opportunity(), false);
     }
 
+    /***
+    * @description Updates first two opportunities in the recurring donation to closed won for calculating rollups
+    */
+    private static void setFewOppsInRdToClosedWonForRollupTests(npe03__Recurring_Donation__c rd) {
```

Comment:
What you think about making this more specific by changing the method signature?
```suggestion
    private static void setRdOppsToClosedWonForRollupTests(npe03__Recurring_Donation__c rd, Integer numOpps) {
```